### PR TITLE
notes_files page improvements some typo fix and old commands for helm…

### PR DESCRIPTION
**Pull request by developer-guy**
…2 changes as helm3 versioned

Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

### CODE REVIEW

File: content/en/docs/chart_template_guide/notes_files.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Your release is named {{ .Release.Name }}.
 To learn more about the release, try:
 
   $ helm status {{ .Release.Name }}
-  $ helm get {{ .Release.Name }}
+  $ helm get notes {{ .Release.Name }}
```
Comment (hickeyma) [inline content/en/docs/chart_template_guide/notes_files.md]: helm get notes  --> helm get all

File: content/en/docs/chart_template_guide/notes_files.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Your release is named rude-cardinal.
 To learn more about the release, try:
 
   $ helm status rude-cardinal
-  $ helm get rude-cardinal
+  $ helm get notes rude-cardinal
```
Comment (hickeyma) [inline content/en/docs/chart_template_guide/notes_files.md]: helm get notes --> helm get all

File: content/en/docs/chart_template_guide/notes_files.md (hunk 1 above)
Comment (developer-guy) [reply content/en/docs/chart_template_guide/notes_files.md]: @hickeyma ok , i fixed that.

File: content/en/docs/chart_template_guide/notes_files.md (hunk 2 above)
Comment (developer-guy) [reply content/en/docs/chart_template_guide/notes_files.md]: @hickeyma ok , i fixed that.

**Pull request merged**